server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/ScheduleController.java

```diff
@@ -61,99 +66,145 @@ public class ScheduleController {
-            @RequestParam(required = false, name = "connectionId") List<Long> connectionIds,
-            @RequestParam(required = false, name = "id") Long id,
-            @RequestParam(required = false, name = "status") List<ScheduleStatus> status,
-            @RequestParam(required = false, name = "type") JobType type,
-            @RequestParam(required = false, name = "startTime") Date startTime,
-            @RequestParam(required = false, name = "endTime") Date endTime,
-            @RequestParam(required = false, name = "creator") String creator,
-            @RequestParam(required = false, name = "projectId") Long projectId) {
 
-        QueryScheduleParams req = QueryScheduleParams.builder()
-                .id(id)
-                .connectionIds(connectionIds)
-                .statuses(status)
-                .type(type)
-                .startTime(startTime)
-                .endTime(endTime)
-                .creator(creator)
-                .projectId(projectId)
-                .build();
-        return Responses.paginated(scheduleService.list(pageable, req));
+    // change log
 
+    @RequestMapping("/schedules/{id:[\\d]+}/changes")
+    public ListResponse<ScheduleChangeLog> listChangeLog(@PathVariable Long id) {
+        throw new UnsupportedException();
     }
 
-    @RequestMapping(value = "/scheduleConfigs/{id:[\\d]+}", method = RequestMethod.GET)
-    public SuccessResponse<ScheduleDetailResp> detailSchedule(@PathVariable Long id) {
-        return Responses.single(scheduleService.getById(id));
+    @RequestMapping("/schedules/{id:[\\d]+}/changes/{scheduleChangeLogId:[\\d]+}")
+    public SuccessResponse<ScheduleChangeLog> getChangeLog(@PathVariable Long id,
+            @PathVariable Long scheduleChangeLogId) {
+        throw new UnsupportedException();
     }
 
-    @RequestMapping(value = "/scheduleConfigs/{scheduleId:[\\d]+}/scheduleTask/{scheduleTaskId:[\\d]+}",
-            method = RequestMethod.GET)
-    public SuccessResponse<ScheduleTaskResp> detailScheduleTask(@PathVariable Long scheduleId,
-            @PathVariable Long scheduleTaskId) {
-        return Responses.single(scheduleService.detailScheduleTask(scheduleId, scheduleTaskId));
+    // schedule task
+
+    @RequestMapping(value = "/schedules/{scheduleId:[\\d]+}/tasks/{taskId:[\\d]+}/executions/latest/terminate",
+            method = RequestMethod.POST)
+    public void terminateTask(@PathVariable Long scheduleId, @PathVariable Long taskId) {
+        throw new UnsupportedException();
     }
 
-    @RequestMapping(value = "/{id:[\\d]+}/jobs/async/batchGetDownloadUrl", method = RequestMethod.POST)
-    public ListResponse<String> getDownloadUrl(@PathVariable Long id, @RequestBody List<String> objectId) {
-        return Responses.list(scheduleService.getAsyncDownloadUrl(id, objectId));
+    @RequestMapping(value = "/schedules/{scheduleId:[\\d]+}/tasks/{taskId:[\\d]+}/executions/latest/stop",
```

Comment:
Is `execution` considered as a resource?

Reply:
The "executions" represents execution records. A task can be executed multiple times, for instance, in cases of retrying or resuming after a pause.